action.yml

```diff
@@ -258,15 +258,15 @@ runs:
             export NIX_INSTALLER_URL=${{ inputs.nix-installer-url }}
           else
             if [ -n "${{ inputs.nix-installer-pr }}" ]; then
-              export NIX_INSTALLER_URL=https://install.determinate.systems/nix/pr/${{ inputs.nix-installer-pr }}
+              export NIX_INSTALLER_URL=https://install.determinate.systems/nix/pr/${{ inputs.nix-installer-pr }}?ci=github
```

Comment:
We'll want to quote these. Sometimes things get hinky with `?`.

Reply:
Right!!!!